### PR TITLE
Pyright

### PR DESCRIPTION
**Pull request by Sean1572**
Ran pyright and added to github actions

Note: Changes need to be tested

### CODE REVIEW

**Comment by Sean1572** [comment]
Seperated models.py into seperate files btw to fix werid import issues with pyright

File: classification/train.py [hunk 1]
```diff
@@ -226,14 +235,14 @@ def valid(model: Any,
     })
 
     print(f"Validation Loss:\t{running_loss/len(data_loader)} \n Validation mAP:\t{valid_map}" )
-    if valid_map > best_valid_map:
-        path = os.path.join("models",wandb_run.name + '.pt')
+    if valid_map > best_valid_map and wandb.run is not None:
```
Comment (sprestrelski) [inline classification/train.py]: should we add an option for saving if people aren't running it with wandb? for people looking to use this package or us in the future for testing

File: classification/train.py [hunk 2]
```diff
@@ -21,24 +21,31 @@
...
+
+from models.timm_model import TimmModel
+from models.early_stopper import EarlyStopper
 
 
 tqdm.pandas()
 time_now  = datetime.datetime.now().strftime('%Y%m%d-%H%M') 
 device = 'cuda' if torch.cuda.is_available() else 'cpu'
-wandb_run = None
+#wandb_run = wandb.run
```
Comment (sprestrelski) [inline classification/train.py]: can we remove this entirely

**Review by sprestrelski — APPROVED** [review]

File: classification/train.py (hunk 1 above)
Comment (benjamin-cates) [reply classification/train.py]: This got fixed in the base branch so I'll just fix it in a merge commit

File: classification/train.py (hunk 2 above)
Comment (benjamin-cates) [reply classification/train.py]: This got fixed in the base branch so I'll just fix it in a merge commit

**Pull request merged**